cloudmarker/events/azrdbmssslenforcementevent.py

```diff
@@ -0,0 +1,104 @@
+"""Microsoft Azure RDBMS SSL Enforcement Event.
+
+This module defines the :class:`AzRDBMSSLEnforcementEvent` class that
+identifies RDBMS servers which have SSL connection enforcement
+disabled. This plugin works on the  properties found in the ``ext``
+bucket of ``rdbms`` records.
+"""
+
+
+import logging
+
+from cloudmarker import util
+
+_log = logging.getLogger(__name__)
+
+
+class AzRDBMSSSLEnforcementEvent:
+    """Az RDBMS SSL enforcement event plugin."""
+
+    def __init__(self):
+        """Create an instance of :class:`AzRDBMSSLEnforcementEvent`."""
+
+    def eval(self, record):
+        """Evaluate Azure RDBMS servers for SSL connection enforcement.
+
+        Arguments:
+            record (dict): A RDBMS record.
+
+        Yields:
+            dict: An event record representing an RDBMS server where SSL
+            connection enforcement is disabled
+
+        """
+        com = record.get('com', {})
+        if com is None:
+            return
+
+        if com.get('record_type') != 'rdbms':
+            return
+
+        if com.get('cloud_type') != 'azure':
+            return
+
+        ext = record.get('ext', {})
+        if ext is None:
+            return
+
+        if ext.get('record_type') not in ['mysql_server',
```

Comment:
If you decide to make this a generic plugin, then this check would change to `com.get('record_type') != 'rdbms'`.

Reply:
Removed the logic which checks for 'record_type' in the `ext` bucket and matches it against certain RDBMS types.